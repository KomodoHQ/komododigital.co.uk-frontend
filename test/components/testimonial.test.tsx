import React from 'react';
import Testimonial from '../../src/components/testimonial';
import renderer from 'react-test-renderer';

test('Testimonial element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<Testimonial data={data} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
