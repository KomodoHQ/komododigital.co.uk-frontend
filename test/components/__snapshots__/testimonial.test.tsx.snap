// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial element is rendered 1`] = `
<article
  className="komodoGridWrapper testimonial-wrapper"
  style={
    Object {
      "background": "linear-gradient(0deg, #0F1E65 0%, #04104A 100%)",
    }
  }
>
  <div
    className="Testimonial"
  >
    <span
      className="name"
    >
      :
    </span>
    <q />
    <hr />
    <span
      className="job"
    />
    <span
      className="company"
    />
  </div>
</article>
`;
